src/components/LocationPage/ShareButtons.style.js

```diff
@@ -0,0 +1,109 @@
+import styled, { css } from 'styled-components';
+import palette from 'assets/theme/palette';
+import { StyledShareButtonStyles } from '../ShareBlock/ShareBlock.style';
+import { COLOR_MAP } from 'common/colors';
+
+const WrapperStyles = css`
+  display: block;
+  position: relative;
+  height: 0;
+  z-index: 1;
+`;
+
+export const DesktopButtonsWrapper = styled.div`
+  display: none;
+
+  @media (min-width: 600px) {
+    ${WrapperStyles};
+    top: 4.875rem;
+    margin-top: ${({ isFirst, chartAbove }) =>
+      !isFirst && chartAbove && '-24px'};
```

Comment:
I think you could maybe get rid of `isFirst` and just rely on `chartAbove`?  (i.e. if chartAbove is true, then isFirst will always be false, so you don't need that check and can drop the `isFirst` property?)

Reply:
Rearranged in ChartsHolder and removed the need for isFirst and chartAbove